app.py

```diff
@@ -293,7 +293,7 @@ def __add_write_node_commands(self, context=None):
                 pn
             )
             self.engine.register_command(
-                "%s [Shotgun]" % profile_name,
+                "%s [ShotGrid]" % profile_name,
```

Comment:
The command name shouldn't change. If we do the people's scripts may break.
What we should do it keep the same name, but set the title to something else so the right name show's up in the menu.
See documentation for [`register_command`](https://developer.shotgridsoftware.com/tk-core/platform.html#sgtk.platform.Engine.register_command)